legend-engine-pure-code-compiled-core/src/main/resources/core/pure/graphFetch/domain/domainManagement.pure

```diff
@@ -125,7 +125,7 @@ function <<access.private>> meta::pure::graphFetch::domain::findGetAllInQualifie
     p | if ($p->instanceOf(QualifiedProperty),
           | let exp =  $p->cast(@QualifiedProperty<Any>).expressionSequence->evaluateAndDeactivate()->last()->toOne();
             let val = $exp->lookForGetAll();
-            if ($val->instanceOf(SimpleFunctionExpression) && $val->cast(@SimpleFunctionExpression).func == getAll_Class_1__T_MANY_,
+            if ($val->instanceOf(SimpleFunctionExpression) && ($val->cast(@SimpleFunctionExpression).func == getAll_Class_1__T_MANY_ || $val->cast(@SimpleFunctionExpression).func == getAll_Class_1__Date_1__Date_1__T_MANY_),
```

Comment:
How we selected these getAll functions?  Seems incomplete.    Look at this function that we might want to see if we can re-use: https://github.com/finos/legend-engine/blob/2739eaf86a70504e5b6360ac0da0f1358c152cd9/legend-engine-pure-code-compiled-core/src/main/resources/core/pure/router/routing/router_routing.pure#L567

Reply:
Valid point, changed it